### DISCUSSION TITLE
Is it possible an Appimage of Audacious?

### DISCUSSION THREAD

**Discussion by solusme** [audacious-media-player/audacious#1386] [Ideas]
Is it possible an Appimage of Audacious?

**Comment by radioactiveman** [comment]
We (the Audacious developers) do not distribute an AppImage of Audacious. The same applies for Flatpak.

However there is a Flatpak version [here](https://flathub.org/apps/org.atheme.audacious) which is maintained by the Flathub community. See also its [GitHub repo](https://github.com/flathub/org.atheme.audacious) for more details.

If you run into issues with it, we can't give support though. We therefore recommend to install Audacious via traditional package sources of your distribution of choice.
